cid/helpers/s3.py

```diff
@@ -18,6 +18,9 @@ def ensure_bucket(self, name: str) -> str:
             response = self.client.head_bucket(Bucket=name)
             return name
         except botocore.exceptions.ClientError as ex:
+            if int(ex.response['Error']['Code']) != 404:
```

Comment:
int is potentially dangerous operation. 
i would do `exc.response['Error']['Code'] != '404':`

or even 
`exc.response.get('Error', {}).get('Code') != '404':` but it is probably too much